sheetstorm/management/commands/generate_test_data.py

```diff
@@ -26,7 +34,13 @@ def handle(self, *args, **options):
         if is_need_to_create_superuser:
             self.create_user(superuser_user_type)
 
+        project_options = self._get_project_options(options)
+
+        for (project_type, number_of_projects) in project_options.items():
+            self.create_project(project_type, number_of_projects)
```

Comment:
My proposal is to extract it also to separate instance method instead of keeping more and more things in `handle()`

Reply:
Done